app/templates/partials/summary/checkbox.html

```diff
@@ -1,5 +1,21 @@
+{% set not_answered = 'No answer provided' %}
 <ul class="u-m-no">
-  {% for selected_answer in answer.value %}
-    <li>{{selected_answer}}</li>
-  {% endfor %}
+      {% for selected_answer in answer.value %}
+        <li>{{selected_answer[0]}}
+        {%- if answer.child_answer_value is none -%}
+            {%- else -%}
+              {% if selected_answer[1] %}
```

Comment:
The 3 lines above can probably be 1?
something like
`{%- if answer.child_answer_value is not none and selected_answer[1] %}`?